src/components/SurveyComponents/EngineeringActivities.tsx

```diff
@@ -0,0 +1,85 @@
+import { FormSelect, Navigation, SurveyStep } from '@components'
+import { useEffect, useRef } from 'react'
+
+const activitiesQuestions = {
+  title: 'Activities',
+  codingPercentage: {
+    type: 'MCQ',
+    question: 'What percentage of your week is spent in coding/testing/implementing?',
+    choices: ['0-20%', '20-40%', '40-60%', '60-80%'],
+    isRequired: false,
+  },
+  designingPercentage: {
+    type: 'MCQ',
+    question: 'What percentage of your week is spent in designing?',
+    choices: ['0-20%', '20-40%', '40-60%', '60-80%'],
+    isRequired: false,
+  },
+  meetingsPercentage: {
+    type: 'MCQ',
+    question: 'What percentage of your week is spent in meetings?',
+    choices: ['0-20%', '20-40%', '40-60%', '60-80%'],
+    isRequired: false,
+  },
+  mentoringPercentage: {
+    type: 'MCQ',
+    question: 'What percentage of your week is spent in mentoring?',
+    choices: ['0-20%', '20-40%', '40-60%', '60-80%'],
+    isRequired: false,
+  },
+}
+interface propTypes {
+  next: (step: number) => void
+  back: () => void
+}
+export function EngineeringActivities({ next, back }: propTypes) {
+  const titleRef = useRef<HTMLHeadingElement | null>(null)
+  useEffect(() => {
+    // scroll to top
+    titleRef.current?.scrollIntoView({ behavior: 'smooth', block: 'center' })
+  }, [])
```

Comment:
By extracting the logic into a function named `scrollToTop`, the purpose of the code is clear without needing a comment.
```js
const scrollToTop = () => {
  titleRef.current?.scrollIntoView({ behavior: 'smooth', block: 'center' });
};

```